<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Zhicheng Ji's Homepage</title>

  <script src="js/jquery-1.11.2.min.js" type="text/javascript"></script>
  <script src="js/jquery-ui.min.js" type="text/javascript"></script>
<script type="text/javascript" charset="utf8" src="//cdn.datatables.net/1.10.7/js/jquery.dataTables.js"></script>
  <link rel="stylesheet" href="css/jquery-ui.css" />
  <link rel="stylesheet" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.7/css/jquery.dataTables.css">

<script type="text/javascript">
$(document).ready(function(){
	$('#table_software').DataTable({
		"columnDefs": [
   	 		{ className: "dt-center", "targets": [ 0,1,2,3 ]},  {"width": "35%", "targets": 0 }
  		],
		"aaSorting": []
	});
	$('#table_statool').DataTable({
                "columnDefs": [
                        { className: "dt-center", "targets": [ 0,1,2 ]},  {"width": "35%", "targets": 0 }
                ],
                "aaSorting": [], bFilter: false, bInfo: false, bPaginate: false
        });
      $('#tabs').tabs({show: 'fade', hide: 'fade'}); 
});
$(function() {
   $( "#Researchacc" ).tabs({ show: 'fade', hide: 'fade', heightStyle: "content" });
});
</script>

</head>
<body>

<center><h1> Zhicheng Ji </h1></center>

<div class="tabs-centre" id="tabs">
      <ul>
        <li><a href="#About">About</a></li>
        <li><a href="#Research">Research</a></li>
        <li><a href="#Software">Software</a></li>
        <li><a href="#CV">CV</a></li>
        <li><a href="#Contact">Contact</a></li>
      </ul>
      
      <center><div id="About" class="container">   
            <center><img src="images/portrait.jpg" width="30%"></center>
            <p>I am an Assistant Professor in <a href="https://biostat.duke.edu/" target="_blank"> Department of Biostatistics and Bioinformatics</a> at Duke University. <br><br> 
            My primary research interest is developing novel statistical and computational methods for integrating and analyzing big and complex genomic data, especially data from single-cell sequencing. Some examples include the differential analysis of single-cell RNA-sequencing (scRNA-seq) data with multiple samples, joint analysis of paired single-cell RNA and T cell receptor (TCR) sequencing data, pseudotime trajectory inference, and signal recovery for single-cell ATAC-sequencing (scATAC-seq) data. I have been actively applying these methods to various biological and clinical studies, such as immunology and cancer.  <br><br> 
	</p>	    
	<br>
		
      </div></center>
  
<!--
<center><div id="Research" class="container">
<div class="tabs-centre" id="Researchacc">
      <ul>
        <li><a href="#Researchsub1">Big Data Genomics</a></li>
        <li><a href="#Researchsub2">Single-Cell Analysis</a></li>
	<li><a href="#Researchsub3">Collaborational Work</a></li>
      </ul>

<br>

  <div id="Researchsub1">
<br>
      <img src="images/GSCA.jpg" width="35%" height="30%" style="float: left; margin-right: 1.5em">
      <h4 style="text-align:left"><u> GSCA: Gene Set Context Analysis </u></h4>
      <p style="text-align:left">
      [<a href="https://github.com/zji90/GSCA" target="_blank">Paper</a>]
      [<a href="https://github.com/zji90/GSCA" target="_blank">Github</a>]
      [<a href="http://www.bioconductor.org/packages/release/bioc/html/GSCA.html" target="_blank">Bioconductor</a>]
      [<a href="https://zhiji.shinyapps.io/GSCA/" target="_blank">Launch Online</a>]
      [<a href="https://www.youtube.com/watch?v=wqv_dmlxdcI" target="_blank">Video Demo</a>]
      [<a href="http://zji90.github.com/GSCA-presentation" target="_blank">Presentation</a>]
      <br>
      GSCA is an open source software package to utilize massive amounts of publicly available gene expression data to make new biological discoveries. Users can interactively visualize transcriptional activities of gene sets in 25,000+ human and mouse samples which represent a broad spectrum of biological contexts. Given one or multiple gene sets, GSCA can query the expression compendium to systematically identify biological contexts associated with highly arbitrary and complex gene set activity patterns. GSCA is a parallel project of GSPA.

</div>

<div id="Researchsub2">
<br>
<img src="images/TSCAN.jpg" width="35%" height="30%" style="float: left; margin-right: 1.5em">
      <h4><u> TSCAN: Tools for Single-Cell ANalysis </u></h4>
      <p>
      [<a href="https://github.com/zji90/TSCAN" target="_blank">Github</a>]
      [<a href="http://www.bioconductor.org/packages/release/bioc/html/TSCAN.html" target="_blank">Bioconductor</a>]
      [<a href="https://zhiji.shinyapps.io/TSCAN" target="_blank">Launch Online</a>]
      [<a href="https://www.youtube.com/watch?v=zdcBAVe1GBE" target="_blank">Video Demo</a>]
      [<a href="http://zji90.github.com/TSCAN-presentation" target="_blank">Presentation</a>]
      <br>
      TSCAN provides a variety of functions to explore single-cell RNA-seq data and construct and tune pseudo-time cell ordering using a new algorithm called hierarchical minimum-spanning-tree (MST). Prior biological information such as expressions of marker genes can be easily incorporated to refine the ordering. TSCAN also allows users to quantitatively compare different pseudo time cell orderings given cell sub-population information.  
     
<br><br><br><br>
        <hr>

 <img src="images/dropout.png" width="35%" height="35%" style="float: left; margin-right: 1.5em; margin-bottom:10%">
      <h4 style="text-align:left"><u> Adjusting drop-out events in single-cell RNA-seq data </u></h4>
      <p style="text-align:left">
      An empirical Bayesian model is derived to adjust the drop-out events in single-cell RNA-seq data. Non-parametric generalized linear models are used to model the drop-out probabilities for the individual gene and cell. After adjustment, the averaged single-cell expressions show higher correlation with bulk expressions compared to raw single-cell expressions. The adjustment is crucial for downstream analysis like differential analysis and pseudo time reconstruction.


</div>

<div id="Researchsub3">
Chi Van Dang Lab @ Upenn
<br>
Andrew Feinberg Lab @ JHU
<br>
Loyal Goff Lab @ JHU
<br>
Sean Taverna Lab @ JHU
<br>
Steven Vokes Lab @ UT Austin
<br>
John Wherry Lab @ Upenn

</div>

</div>
  </div>
</center>
-->

      
      <center><div id="Research" class="container">
<br>
<b>1. Methods for analyzing single-cell genomic data</b>
<br>
<br>
TSCAN: Pseudo-time reconstruction and evaluation in single-cell RNA-seq analysis [<a href="http://nar.oxfordjournals.org/content/44/13/e117" target="_blank">Paper</a>][<a href="https://github.com/zji90/TSCAN" target="_blank">Software</a>]
<br>
<br>
SCRAT: Single-cell regulome data analysis [<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx315" target="_blank">Paper</a>][<a href="https://github.com/zji90/SCRAT" target="_blank">Software</a>]
<br>
<br>
SCATE: Single-cell ATAC-seq signal extraction and enhancement [<a href="https://www.biorxiv.org/content/10.1101/795609v1" target="_blank">Paper</a>][<a href="https://github.com/zji90/SCATE" target="_blank">Software</a>]
<br>
<br>
STIP: State transition inference prediction [<a href="https://www.cell.com/immunity/fulltext/S1074-7613(19)30409-1" target="_blank">Paper</a>][<a href="https://github.com/zji90/STIP" target="_blank">Software</a>]
<br>
<br>
BIRD: Global prediction of chromatin accessibility using small-cell-number and single-cell RNA-seq [<a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz716/5552071" target="_blank">Paper</a>][<a href="https://github.com/WeiqiangZhou/BIRD" target="_blank">Software</a>]
<br>
<br>
<b>2. Methods for analyzing large-scale multi-modal genomic data</b>
<br>
<br>
GSCA: Gene Set Context Analysis for Turning Publicly Available Gene Expression Data into Discoveries. [<a href="http://nar.oxfordjournals.org/content/44/1/e8" target="_blank">Paper</a>][<a href="https://github.com/zji90/GSCA" target="_blank">Software</a>]
<br>
<br>
DynaMO: Dynamic motif occupancy analysis [<a href="https://academic.oup.com/nar/article/46/1/e2/4372507" target="_blank">Paper</a>][<a href="https://github.com/spo111/DynaMO" target="_blank">Software</a>]
<br>
<br>
<br>
<a href="https://scholar.google.com/citations?user=Uih8yHsAAAAJ&hl=en" target="_blank">All Publications on Google Scholar</a>
 
</div></center>
      
      <center><div id="Software" class="container">        
	<table id="table_software" class="display">
    <thead>
        <tr>
            <th>Name</th>
            <th>Type</th>
	<th>Author</th>
	<th>Availability</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>SCATE: Single-cell ATAC-seq signal extraction and enhancement</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SCATE" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>TSCAN: Tools for Single-Cell ANalysis</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/TSCAN" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/TSCAN.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/TSCAN/" target="_blank">[Online GUI]</a></td>
        </tr>
	<tr>
            <td>BIRD: Big data regression for predicting DNase I hypersensitivity</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/WeiqiangZhou/BIRD" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>SCRAT: Single-Cell Regulome Analysis Tool</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Weiqiang Zhou, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SCRAT" target="_blank">[Github]</a>&nbsp<a href="https://zhiji.shinyapps.io/SCRAT/" target="_blank">[Online GUI]</a></td>
        </tr>
	<tr>
            <td>STIP: State transition inference prediction</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/STIP" target="_blank">[Github]</a></td>
        </tr>
        <tr>
            <td>GSCA: Gene Set Context Analysis</td>
            <td>R Software Package</td>
	<td>Zhicheng Ji, Hongkai Ji</td>
	<td><a href="https://github.com/zji90/GSCA" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/GSCA.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/GSCA/" target="_blank">[Online GUI]</a></td>
        </tr>
        <tr>
            <td>RMRCE: Regularized Maximum Rank Correlation Estimator</td>
            <td>R Software Package</td>
        <td>Fang Han, Hongkai Ji, Zhicheng Ji, Honglang Wang</td>
        <td><a href="https://github.com/zji90/RMRCE" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>SEPA: Single-cell Gene Expression Pattern Analysis</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SEPA" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/SEPA.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/SEPA/" target="_blank">[Online GUI]</a></td>
        </tr> 
    	<tr>
            <td>GEOsearch: Extendable Search Engine for GEO</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/GEOsearch" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/GEOsearch.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/GEOsearch/" target="_blank">[Online GUI]</a></td>
        </tr>
        <tr>
            <td>Reproducible Interactive Data Exploration Tool</td>
            <td>Web Application</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://zhiji.shinyapps.io/ixplore/" target="_blank">[Online GUI]</a></td>
        </tr>
	</tbody>
</table>

      </div></center>
      
      <div id="CV" class="container" style="margin:0 auto;text-align:center">        
	<object data="https://github.com/zji90/zji90.github.io/raw/master/CV.pdf" type="application/pdf" width=1100px height=1000px style="margin-left:auto;margin-right:auto;text-align:center">
	<p>It appears you don't have a PDF plugin for this browser. <a href="http://www.biostat.jhsph.edu/~zji4/CV021920.pdf" target="_blank">Click here to download the PDF file.</a></p>
</object>
      </div>
       
<!--	
      <center><div id="Statisticstoolbox" class="container">        
<center><h3> Statistics Toolbox: Easy Statistics for Non Programmers </h3></center>

Statistics Toolbox is a quick and convenient tool for users to perform common tasks in statistics without writing computer programs. Statistics Toolbox currently provides three major statistics functions: 1. drawing and calculating pdf and cdf functions of common statistics distributions; 2. Performing common statistical tests; 3. Calculating sample size and statistical power. Statistics Toolbox uses the same algorithms as those used by R. Statistics Toolbox is free on all platforms.
<br>
统计工具箱是一款无需编程即可方便快捷实现多种统计方法的工具。统计工具箱目前提供三种主要统计功能：1. 常用统计分布概率密度函数和累积分布函数的作图及计算；2. 常用统计检验；3. 样本量及统计效力计算。统计工具箱使用与R语言一致的算法。统计工具箱在所有平台上免费。
<br>
<br> 
Statistics Toolbox is currently available on following platforms:
<br>
统计工具箱现可在如下平台获取： 
<br>
<br>
<br>
      <center>
        <table id="table_statool" class="display">
    <thead>
        <tr>
            <th>Platform 平台</th>
            <th>Language 语言</th>
		<th>Version 版本</th>
        <th>Link 链接</th>
        </tr>
    </thead>
	<tbody>
        <tr>
            <td>ios (iphone & ipad)</td>
            <td>English</td>
	<td>1.3</td>
        <td><a href="https://itunes.apple.com/us/app/statistics-toolbox/id1016200287" target="_blank">itunes</a></td>
        </tr>
	<tr>
		<td>Online User Interface</td>
		<td>English</td>
		<td>1.0</td>
		<td><a href="https://statool.shinyapps.io/English" target="_blank">Online GUI</a></td>
	</tr>
	<tr>
                <td>Online User Interface 在线用户界面</td>
                <td>Chinese 中文</td>
		<td>1.0</td>
                <td><a href="https://statool.shinyapps.io/Chinese" target="_blank">Online 
GUI</a></td>
        </tr>
	<tr>
                <td>R Package R 程序包</td>
                <td>English and Chinese 英文和中文</td>
		<td>1.0</td>
                <td><a href="https://github.com/zji90/statisticstoolbox" target="_blank">Github</a></td>
        </tr>
	</tbody>
</table>
<br> 
<br> 
	Please make donations to this project using the following Paypal link. Thanks!
	<br>	
	欢迎通过如下Paypal链接对此项目进行捐赠！非常感谢！
	<br><br>	

<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
<input type="hidden" name="cmd" value="_s-xclick">
<input type="hidden" name="encrypted" value="-----BEGIN PKCS7-----MIIHNwYJKoZIhvcNAQcEoIIHKDCCByQCAQExggEwMIIBLAIBADCBlDCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb20CAQAwDQYJKoZIhvcNAQEBBQAEgYC9GrjO9VRzxDr0zWFIDVlTl3Tl7ZJmOrfdQRq6WoxT6rAylfZGmCZ/GH6kRdK1IQ72R6e/I0PzhuKbZhSEqcC74Na32UlSDC7UKxfFxJAawy42LDes4f9Jy98H76Ucm6yZKSBNPMoniYsd1ccFe/fsXIz4n8per1XvdYwaOKayDzELMAkGBSsOAwIaBQAwgbQGCSqGSIb3DQEHATAUBggqhkiG9w0DBwQIJ/whLeMQyxSAgZBewMXVrsS+AjZLXe8O4tBXUMBNcgZMz7Kigws+vM7+8y4M87/FfLGRSysIp8HRo83PrZL7tZjyGm8N3xW/c8qo65cd4Moc5W+FJ145EDmIH+HRWnbIiTDbtU6H3RBu2uDhLAcohmxjVXfB7h+oWzZNe5pWSTxgcEFqWqylEP2JwUKspiyvDTG3bYL6TjaVU3CgggOHMIIDgzCCAuygAwIBAgIBADANBgkqhkiG9w0BAQUFADCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb20wHhcNMDQwMjEzMTAxMzE1WhcNMzUwMjEzMTAxMzE1WjCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb20wgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAMFHTt38RMxLXJyO2SmS+Ndl72T7oKJ4u4uw+6awntALWh03PewmIJuzbALScsTS4sZoS1fKciBGoh11gIfHzylvkdNe/hJl66/RGqrj5rFb08sAABNTzDTiqqNpJeBsYs/c2aiGozptX2RlnBktH+SUNpAajW724Nv2Wvhif6sFAgMBAAGjge4wgeswHQYDVR0OBBYEFJaffLvGbxe9WT9S1wob7BDWZJRrMIG7BgNVHSMEgbMwgbCAFJaffLvGbxe9WT9S1wob7BDWZJRroYGUpIGRMIGOMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExFjAUBgNVBAcTDU1vdW50YWluIFZpZXcxFDASBgNVBAoTC1BheVBhbCBJbmMuMRMwEQYDVQQLFApsaXZlX2NlcnRzMREwDwYDVQQDFAhsaXZlX2FwaTEcMBoGCSqGSIb3DQEJARYNcmVAcGF5cGFsLmNvbYIBADAMBgNVHRMEBTADAQH/MA0GCSqGSIb3DQEBBQUAA4GBAIFfOlaagFrl71+jq6OKidbWFSE+Q4FqROvdgIONth+8kSK//Y/4ihuE4Ymvzn5ceE3S/iBSQQMjyvb+s2TWbQYDwcp129OPIbD9epdr4tJOUNiSojw7BHwYRiPh58S1xGlFgHFXwrEBb3dgNbMUa+u4qectsMAXpVHnD9wIyfmHMYIBmjCCAZYCAQEwgZQwgY4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEUMBIGA1UEChMLUGF5UGFsIEluYy4xEzARBgNVBAsUCmxpdmVfY2VydHMxETAPBgNVBAMUCGxpdmVfYXBpMRwwGgYJKoZIhvcNAQkBFg1yZUBwYXlwYWwuY29tAgEAMAkGBSsOAwIaBQCgXTAYBgkqhkiG9w0BCQMxCwYJKoZIhvcNAQcBMBwGCSqGSIb3DQEJBTEPFw0xNTA4MjIxMDM3MDhaMCMGCSqGSIb3DQEJBDEWBBQ79TI2OgqKR3XY8HuCuxBoBCHhrjANBgkqhkiG9w0BAQEFAASBgCkPEU13GeGiUUYDT2wBj3wGM9zS/Hp8UkiG8d62hnPS9jx6jj/a8kAO+UjA3DUqm/9PvpCTRLNPIRMbMjA/b+hQrznL/0v+Af4PzHWXsTo7olUcm4n7G9iOPqUkNH3MTmguF7n7G6ttzRtvvm/6c1PeaHk0Adb5TjwjQoUbiTgr-----END PKCS7-----
">
<input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online!">
<img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
</form>
<br>
<br>
Statistics Toolbox on all platforms is designed, developed and maintained by Zhicheng Ji. All rights reserved.
 </center>
      </div></center>
-->


      <center><div id="Contact" class="container">        
            <p>Office: <br>
            E3038 Department of Biostatistics <br>
            Bloomberg School of Public Health <br>
            615 North Wolfe Street <br> 
            Baltimore, MD, 21205, USA </p>
            <p>Email: <br><a href="mailto:zji4@jhu.edu">zji4@jhu.edu</a></p>
            <p>Github: <br><a href="https://github.com/zji90" target="_blank">https://github.com/zji90</a></p>
      </div></center>
    </div> 

 
</body>
</html>


