<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Zhicheng Ji's Homepage</title>

  <script src="js/jquery-1.11.2.min.js" type="text/javascript"></script>
  <script src="js/jquery-ui.min.js" type="text/javascript"></script>
<script type="text/javascript" charset="utf8" src="//cdn.datatables.net/1.10.7/js/jquery.dataTables.js"></script>
  <link rel="stylesheet" href="css/jquery-ui.css" />
  <link rel="stylesheet" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.7/css/jquery.dataTables.css">

<script type="text/javascript">
$(document).ready(function(){
	$('#table_software').DataTable({
		"columnDefs": [
   	 		{ className: "dt-center", "targets": [ 0,1,2,3 ]},  {"width": "35%", "targets": 0 }
  		],
		"aaSorting": []
	});
	$('#table_statool').DataTable({
                "columnDefs": [
                        { className: "dt-center", "targets": [ 0,1,2 ]},  {"width": "35%", "targets": 0 }
                ],
                "aaSorting": [], bFilter: false, bInfo: false, bPaginate: false
        });
      $('#tabs').tabs({show: 'fade', hide: 'fade'}); 
});
$(function() {
   $( "#Researchacc" ).tabs({ show: 'fade', hide: 'fade', heightStyle: "content" });
});
</script>

</head>
<body>

<center><h1> Zhicheng Ji </h1></center>

<div class="tabs-centre" id="tabs">
      <ul>
        <li><a href="#About">About</a></li>
        <li><a href="#Research">Research</a></li>
        <li><a href="#Software">Software</a></li>
        <li><a href="#CV">CV</a></li>
        <li><a href="#Contact">Join</a></li>
      </ul>
      
      <center><div id="About" class="container">   
            <img src="images/portrait.jpg" width="30%" style="float:right">
            <p>I am an Assistant Professor in <a href="https://biostat.duke.edu/" target="_blank"> Department of Biostatistics and Bioinformatics</a> at Duke University. <br><br> 
            My primary research interest is developing novel statistical and computational methods for integrating and analyzing big and complex genomic data, especially data from single-cell sequencing. Some examples include the differential analysis of single-cell RNA-sequencing (scRNA-seq) data with multiple samples, joint analysis of paired single-cell RNA and T cell receptor (TCR) sequencing data, pseudotime trajectory inference, and signal recovery for single-cell ATAC-sequencing (scATAC-seq) data. I have been actively applying these methods to various biological and clinical studies, such as immunology and cancer.  <br><br> 
	</p>	    
	
		I am looking for students and postdocs to work on exciting research projects. Please contact me if your are interested. Email: zhicheng.ji@duke.edu
		
		
		
      </div></center>
  
<!--
<center><div id="Research" class="container">
<div class="tabs-centre" id="Researchacc">
      <ul>
        <li><a href="#Researchsub1">Big Data Genomics</a></li>
        <li><a href="#Researchsub2">Single-Cell Analysis</a></li>
	<li><a href="#Researchsub3">Collaborational Work</a></li>
      </ul>

<br>

  <div id="Researchsub1">
<br>
      <img src="images/GSCA.jpg" width="35%" height="30%" style="float: left; margin-right: 1.5em">
      <h4 style="text-align:left"><u> GSCA: Gene Set Context Analysis </u></h4>
      <p style="text-align:left">
      [<a href="https://github.com/zji90/GSCA" target="_blank">Paper</a>]
      [<a href="https://github.com/zji90/GSCA" target="_blank">Github</a>]
      [<a href="http://www.bioconductor.org/packages/release/bioc/html/GSCA.html" target="_blank">Bioconductor</a>]
      [<a href="https://zhiji.shinyapps.io/GSCA/" target="_blank">Launch Online</a>]
      [<a href="https://www.youtube.com/watch?v=wqv_dmlxdcI" target="_blank">Video Demo</a>]
      [<a href="http://zji90.github.com/GSCA-presentation" target="_blank">Presentation</a>]
      <br>
      GSCA is an open source software package to utilize massive amounts of publicly available gene expression data to make new biological discoveries. Users can interactively visualize transcriptional activities of gene sets in 25,000+ human and mouse samples which represent a broad spectrum of biological contexts. Given one or multiple gene sets, GSCA can query the expression compendium to systematically identify biological contexts associated with highly arbitrary and complex gene set activity patterns. GSCA is a parallel project of GSPA.

</div>

<div id="Researchsub2">
<br>
<img src="images/TSCAN.jpg" width="35%" height="30%" style="float: left; margin-right: 1.5em">
      <h4><u> TSCAN: Tools for Single-Cell ANalysis </u></h4>
      <p>
      [<a href="https://github.com/zji90/TSCAN" target="_blank">Github</a>]
      [<a href="http://www.bioconductor.org/packages/release/bioc/html/TSCAN.html" target="_blank">Bioconductor</a>]
      [<a href="https://zhiji.shinyapps.io/TSCAN" target="_blank">Launch Online</a>]
      [<a href="https://www.youtube.com/watch?v=zdcBAVe1GBE" target="_blank">Video Demo</a>]
      [<a href="http://zji90.github.com/TSCAN-presentation" target="_blank">Presentation</a>]
      <br>
      TSCAN provides a variety of functions to explore single-cell RNA-seq data and construct and tune pseudo-time cell ordering using a new algorithm called hierarchical minimum-spanning-tree (MST). Prior biological information such as expressions of marker genes can be easily incorporated to refine the ordering. TSCAN also allows users to quantitatively compare different pseudo time cell orderings given cell sub-population information.  
     
<br><br><br><br>
        <hr>

 <img src="images/dropout.png" width="35%" height="35%" style="float: left; margin-right: 1.5em; margin-bottom:10%">
      <h4 style="text-align:left"><u> Adjusting drop-out events in single-cell RNA-seq data </u></h4>
      <p style="text-align:left">
      An empirical Bayesian model is derived to adjust the drop-out events in single-cell RNA-seq data. Non-parametric generalized linear models are used to model the drop-out probabilities for the individual gene and cell. After adjustment, the averaged single-cell expressions show higher correlation with bulk expressions compared to raw single-cell expressions. The adjustment is crucial for downstream analysis like differential analysis and pseudo time reconstruction.


</div>

<div id="Researchsub3">
Chi Van Dang Lab @ Upenn
<br>
Andrew Feinberg Lab @ JHU
<br>
Loyal Goff Lab @ JHU
<br>
Sean Taverna Lab @ JHU
<br>
Steven Vokes Lab @ UT Austin
<br>
John Wherry Lab @ Upenn

</div>

</div>
  </div>
</center>
-->

      
      <center><div id="Research" class="container">
<br>
<b>1. Methods for analyzing single-cell genomic data</b>
<br>
<br>
TSCAN: Pseudo-time reconstruction and evaluation in single-cell RNA-seq analysis [<a href="http://nar.oxfordjournals.org/content/44/13/e117" target="_blank">Paper</a>][<a href="https://github.com/zji90/TSCAN" target="_blank">Software</a>]
<br>
<br>
SCRAT: Single-cell regulome data analysis [<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx315" target="_blank">Paper</a>][<a href="https://github.com/zji90/SCRAT" target="_blank">Software</a>]
<br>
<br>
SCATE: Single-cell ATAC-seq signal extraction and enhancement [<a href="https://www.biorxiv.org/content/10.1101/795609v1" target="_blank">Paper</a>][<a href="https://github.com/zji90/SCATE" target="_blank">Software</a>]
<br>
<br>
STIP: State transition inference prediction [<a href="https://www.cell.com/immunity/fulltext/S1074-7613(19)30409-1" target="_blank">Paper</a>][<a href="https://github.com/zji90/STIP" target="_blank">Software</a>]
<br>
<br>
BIRD: Global prediction of chromatin accessibility using small-cell-number and single-cell RNA-seq [<a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz716/5552071" target="_blank">Paper</a>][<a href="https://github.com/WeiqiangZhou/BIRD" target="_blank">Software</a>]
<br>
<br>
<b>2. Methods for analyzing large-scale multi-modal genomic data</b>
<br>
<br>
GSCA: Gene Set Context Analysis for Turning Publicly Available Gene Expression Data into Discoveries. [<a href="http://nar.oxfordjournals.org/content/44/1/e8" target="_blank">Paper</a>][<a href="https://github.com/zji90/GSCA" target="_blank">Software</a>]
<br>
<br>
DynaMO: Dynamic motif occupancy analysis [<a href="https://academic.oup.com/nar/article/46/1/e2/4372507" target="_blank">Paper</a>][<a href="https://github.com/spo111/DynaMO" target="_blank">Software</a>]
<br>
<br>
<br>
<a href="https://scholar.google.com/citations?user=Uih8yHsAAAAJ&hl=en" target="_blank">All Publications on Google Scholar</a>
 
</div></center>
 
      <center><div id="Software" class="container">        
	<table id="table_software" class="display">
    <thead>
        <tr>
            <th>Name</th>
            <th>Type</th>
	<th>Author</th>
	<th>Availability</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>SCATE: Single-cell ATAC-seq signal extraction and enhancement</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SCATE" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>TSCAN: Tools for Single-Cell ANalysis</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/TSCAN" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/TSCAN.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/TSCAN/" target="_blank">[Online GUI]</a></td>
        </tr>
	<tr>
            <td>BIRD: Big data regression for predicting DNase I hypersensitivity</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/WeiqiangZhou/BIRD" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>SCRAT: Single-Cell Regulome Analysis Tool</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Weiqiang Zhou, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SCRAT" target="_blank">[Github]</a>&nbsp<a href="https://zhiji.shinyapps.io/SCRAT/" target="_blank">[Online GUI]</a></td>
        </tr>
	<tr>
            <td>STIP: State transition inference prediction</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/STIP" target="_blank">[Github]</a></td>
        </tr>
        <tr>
            <td>GSCA: Gene Set Context Analysis</td>
            <td>R Software Package</td>
	<td>Zhicheng Ji, Hongkai Ji</td>
	<td><a href="https://github.com/zji90/GSCA" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/GSCA.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/GSCA/" target="_blank">[Online GUI]</a></td>
        </tr>
        <tr>
            <td>RMRCE: Regularized Maximum Rank Correlation Estimator</td>
            <td>R Software Package</td>
        <td>Fang Han, Hongkai Ji, Zhicheng Ji, Honglang Wang</td>
        <td><a href="https://github.com/zji90/RMRCE" target="_blank">[Github]</a></td>
        </tr>
	<tr>
            <td>SEPA: Single-cell Gene Expression Pattern Analysis</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/SEPA" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/SEPA.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/SEPA/" target="_blank">[Online GUI]</a></td>
        </tr> 
    	<tr>
            <td>GEOsearch: Extendable Search Engine for GEO</td>
            <td>R Software Package</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://github.com/zji90/GEOsearch" target="_blank">[Github]</a>&nbsp<a href="http://www.bioconductor.org/packages/release/bioc/html/GEOsearch.html" target="_blank">[Bioconductor]</a>&nbsp<a href="https://zhiji.shinyapps.io/GEOsearch/" target="_blank">[Online GUI]</a></td>
        </tr>
        <tr>
            <td>Reproducible Interactive Data Exploration Tool</td>
            <td>Web Application</td>
        <td>Zhicheng Ji, Hongkai Ji</td>
        <td><a href="https://zhiji.shinyapps.io/ixplore/" target="_blank">[Online GUI]</a></td>
        </tr>
	</tbody>
</table>

      </div></center>
      
      <center><div id="CV" class="container">        
      <br>
		<p><a href="https://github.com/zji90/zji90.github.io/raw/master/CV.pdf" target="_blank">Curriculum Vitae</a></p>
 </div></center>

      <center><div id="Contact" class="container">        
            <p>Email: <br><a href="mailto:zhicheng.ji@duke.edu">zhicheng.ji@duke.edu</a></p>
            <p>Github: <br><a href="https://github.com/zji90" target="_blank">https://github.com/zji90</a></p>
      </div></center>
    </div> 

 
</body>
</html>


